packs/feats/arcane-fists.json

```diff
@@ -43,12 +43,10 @@
                 ]
             },
             {
-                "definition": [
-                    "item:trait:unarmed"
-                ],
-                "key": "AdjustStrike",
+                "itemId": "xxxxxxFISTxxxxxx",
+                "key": "ItemAlteration",
                 "mode": "add",
-                "property": "weapon-traits",
+                "property": "traits",
                 "value": "arcane"
```

Comment:
from what I can tell, this particular trait is for all unarmed attacks, not just fist